beluga/include/beluga/sensor/likelihood_field_model.hpp

```diff
@@ -58,6 +58,14 @@ struct LikelihoodFieldModelParam {
    * Used to calculate the probability of the obstacle being hit.
    */
   double sigma_hit = 0.2;
+  /// Pre-computed two_squared_sigma (auxiliar variable)
+  double two_squared_sigma = 2 * sigma_hit * sigma_hit;
```

Comment:
@DPR00 this won't be recomputed if the user instantiates `LikelihoodFieldModelParam` and then sets parameters. Consider moving this over to the `LikelihoodFieldSensor` class instead, no need for this unknown distance gimmick to be exposed to the user.

Reply:
I was thinking about moving it as protected inside the `LikelihoodFieldModel` class, WDYT?

Reply:
Moving it inside the `make_likelihood_field` function, since there are used just there.